node_modules/oae-core/groupprofile/bundles/default.properties

```diff
@@ -0,0 +1,3 @@
+ADD_A_DESCRIPTION_TO_THIS_GROUP = Add a description to this group
+NO_DESCRIPTION_PROVIDED_YET = No description provided yet
```

Comment:
Should the profile pics move upward now that there is no header, or do we keep a blank strip of space where the header would have been? Keeping in mind the activity header to the left.